includes/class-wc-countries.php

```diff
@@ -57,6 +57,17 @@ public function get_countries() {
 		return $this->countries;
 	}
 
+	/**
+	 * Check if a given code represents a valid ISO 3166-1 alpha-2 code for a country known to us.
+	 *
+	 * @since 5.1.0
+	 * @param string $country_code The country code to check as a ISO 3166-1 alpha-2 code.
+	 * @return bool True if the country is known to us, false otherwise.
+	 */
+	public function country_exists( $country_code ) {
+		return array_key_exists( $country_code, $this->get_countries() );
```

Comment:
```suggestion
		return isset( $country_code, $this->get_countries() );
```
Since `isset` is a language construct it is slightly faster than calling the function `array_key_exists`. You only need to use array_key_exists if might be dealing with an array which contains null values. See https://www.php.net/manual/en/function.array-key-exists.php#example-5434

Reply:
Thanks for the suggestion but the actual syntax is slightly different: `isset( $this->get_countries()[ $country_code ] );` - I'll apply it and push by myself.